b2b-sample/package.json

```diff
@@ -40,14 +40,15 @@
     "jest": "28.1.1",
     "jest-environment-jsdom": "28.1.1",
     "less": "^4.1.3",
-    "next-auth": "^4.16.4",
+    "next-auth": "^4.24.7",
     "next-with-less": "^2.0.5",
     "nx": "15.1.1",
     "react-final-form": "^6.5.9",
     "react-test-renderer": "18.2.0",
     "rsuite": "^5.22.2",
     "ts-jest": "28.0.5",
     "ts-node": "10.9.1",
+    "typescript": "^4.9.3",
```

Comment:
if possible bump typescript to the latest as well

Reply:
Typescript doesn't follow semver, so the patch/minor versions can have breaking changes. just FYI to the OP :)

Reply:
Agreed, but what I feel is that if the application does not break it's better to bump the version with this effort.

WDYT?

Reply:
Agreed.

Reply:
Addressed with https://github.com/wso2/samples-is/pull/479/commits/ead0b7971adbfff874ce16737e69794302cd1261. 
No breaking changes observed when building the app or in app's functionality after the bump.